bench/Alice/README.md

```diff
@@ -1,108 +1,76 @@
-   byts::common:     598904,    600095,    600805,    609538 (mean:     602335, median:     600450, r: 4213.47), Winner
-   .net::common:    1516400,   1549349,   1571868,   1691908 (mean:    1582381, median:    1560608, r: 66240.65), 1.6x slower
-short_lines (^.{16,20}$)
-rure::short_lines:     203152,    203469,    203883,    282480 (mean:     223246, median:     203676, r: 34199.75), 807.2x slower
-byts::short_lines:        233,       245,       260,       571 (mean:        327, median:        252, r: 141.05), Winner
-.net::short_lines:        291,       325,       550,     11850 (mean:       3254, median:        437, r: 4963.90), 0.7x slower
-dotplus (.+)
-  rure::dotplus:    1020685,   1024777,   1042517,   1092363 (mean:    1045085, median:    1033647, r: 28502.54), 0.2x slower
-  byts::dotplus:     831937,    833627,    834811,    834894 (mean:     833817, median:     834219, r: 1195.67), Winner
-  .net::dotplus:    1433560,   1472972,   1616928,   1737061 (mean:    1565130, median:    1544950, r: 120464.79), 0.9x slower
-dotplus_nl ((?s).+)
-rure::dotplus_nl:     482049,    482227,    497715,    514100 (mean:     494022, median:     489971, r: 13221.56), 0.8x slower
-byts::dotplus_nl:     277599,    277733,    278067,    280278 (mean:     278419, median:     277900, r: 1086.60), Winner
-.net::dotplus_nl:    1561056,   1642134,   1644580,   1645318 (mean:    1623272, median:    1643357, r: 35939.75), 4.9x slower
-alice_hattter (Alice.{0,25}Hatter|Hatter.{0,25}Alice)
-rure::alice_hattter:     269886,    273955,   1320481,   2234540 (mean:    1024715, median:     797218, r: 819231.57), 8.9x slower
-byts::alice_hattter:      79760,     80411,     81211,     84605 (mean:      81496, median:      80811, r: 1866.68), Winner
-.net::alice_hattter:    2462189,   2944215,   3160557,   3213413 (mean:    2945093, median:    3052386, r: 296486.68), 36.8x slower
-name_suffixes (([A-Za-z]lice|[A-Za-z]heshire)[^a-zA-Z])
-rure::name_suffixes:     660516,    680393,    726359,    830451 (mean:     724429, median:     703376, r: 65704.70), 0.6x slower
-byts::name_suffixes:     401378,    436962,    437661,    442324 (mean:     429581, median:     437311, r: 16413.10), Winner
-.net::name_suffixes:   24839809,  24893901,  26018144,  31502366 (mean:   26813555, median:   25456022, r: 2747652.27), 57.2x slower
-dotstar (.*)
-  rure::dotstar:    1455614,   1456097,   1460706,   1591548 (mean:    1490991, median:    1458401, r: 58090.48), 0.1x slower
-  byts::dotstar:    1269216,   1270864,   1271539,   1273704 (mean:    1271330, median:    1271201, r: 1609.79), Winner
-  .net::dotstar:    2040246,   2053420,   2229392,   2240420 (mean:    2140869, median:    2141406, r: 94232.48), 0.7x slower
-  ```
\ No newline at end of file
+|               Method |  Job | Runtime |              Pattern |         Mean |        Error |       StdDev |       Median | Ratio | RatioSD | Rank | Gen 0/1k Op | Gen 1/1k Op | Gen 2/1k Op | Allocated Memory/Op |
```

Comment:
Can we control the order of these columns? The `Job` and `Runtime` columns also seem to be superflous.

Reply:
Probobally, although I don't have the answer on top of head : )